packages/evolution-backend/src/services/audits/auditChecks/checks/README.md

```
@@ -0,0 +1,166 @@
+# Audit Check Functions
+
+This directory contains the actual audit check functions organized by survey object type.
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)